photonjs/README.md

```diff
@@ -620,6 +623,15 @@ Proposed simpler model:
 
 <img src="./pagination-simplified.png" alt="image-20191229152718492" style="zoom:50%;" />
 
+# Raw database access
+Raw database access is being exposed in a minimal fashion with the `prisma.raw()` method.
+It allows you to execute raw SQL (and later as well different query languages, when NoSQL databases are added) on the database. While this raw database access enables you to work around the limitations of the Prisma Client queries, it's important to note, that this initial version doesn't protect you from SQL injections. So use this with precaution.
```

Comment:
The section on SQL injection is too conversational:

```suggestion
It allows you to execute raw SQL (and later as well different query languages, when NoSQL databases are added) on the database. Prisma accepts any raw SQL string and executes it directly against the database. Prisma does not perform escaping of user input, so this should be handled by the user in order to prevent SQL injection attacks. Numerous libraries exist for this purpose, including the popular `sqlstring ` for MySQL.
```